production_services/settings.py

```diff
@@ -100,7 +100,9 @@
         'HOST': os.getenv('USER_DB_HOST', '127.0.0.1'),
         'PORT': os.getenv('USER_DB_PORT', '3306'),
         'OPTIONS': {
-            "init_command": "SET default_storage_engine=MYISAM",
+            "init_command": "SET default_storage_engine={}".format(
```

Comment:
Actually I just removed completely this statement in another PR :-)

Reply:
Okay, the parametrised charset could be useful though. Should I remove the engine line?

Reply:
Since this is only used when creating the database schema (hence only for automated tests) I would say yes.

Reply:
Reverted in ead117e